### DISCUSSION TITLE
某个组合按键映射无反应

### DISCUSSION THREAD

**Discussion by Rewrite0** [nshen/learn-neovim-lua#150] [Q&A] [answered]
设置分屏窗口跳转的快捷键时，alt + h j l 也就是左右下这三个都能生效，但是alt + k没有反应
排查了没有冲突的键位，下面的leader + k也可以生效
这让我百思不得其解，不知道有没有什么方法来解决这个问题
![image](https://user-images.githubusercontent.com/49576640/205311040-e1a64a37-5f15-41aa-b631-3b65c5e93584.png)

**Comment by nshen** [comment, marked as answer]
是 windows wsl 吗？查一查是不是 windows terminal 自带的快捷键？

**Reply by Rewrite0** [reply]
Arch Linux,用的wayfire,但是wayfire的快捷键并没有alt+k。。

**Reply by nshen** [reply]
如果排除了系统快捷键和终端快捷键的话，可以调用 `:nmap` 可以看看
<img width="899" alt="image" src="https://user-images.githubusercontent.com/181506/205420955-65fd90a3-aaad-4217-89b2-b9c5058bb836.png">

**Reply by Rewrite0** [reply]
找到原因了，我终端有一个alt+k的快捷键，因为太久没用都忘了，啊这。
